src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypeCalculationArgumentCollection.java

```diff
@@ -37,6 +38,12 @@ public GenotypeCalculationArgumentCollection( final GenotypeCalculationArgumentC
     }
 
     /**
+     * Use the new allele frequency / QUAL score model
+     */
+    @Argument(fullName = "useNewAFCalculator", shortName = "newQual", doc = "If provided, we will use the new AF model instead of the so-called exact model", optional = true)
+    public boolean USE_NEW_AF_CALCULATOR = false;
```

Comment:
Ok, I thought that that was the tool you were updating... so what tool you using to execute the new code? HC? The unified approach to select implemenetation would still apply.